File: apps/websocket-skyblock/src/mongodb/collections/Session.cs
using System.Runtime.Serialization;
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Driver;

public class Session
{
    [BsonId]
    [DataMember]
    public MongoDB.Bson.ObjectId _id { get; set; }
    /// <summary>
    /// The date at which the session key expires.
    /// </summary>
    public DateTime ExpiresAt;
    /// <summary>
    /// The primary identifier of the session.
    /// </summary>
    public string ID = "";
    /// <summary>
    /// The discord id the session is for.
    /// </summary>
    public ulong UserId = 0;
    /// <summary>
    /// The username the session is for.
    /// </summary>
    public string Username = "";

    public Session(DateTime _ExpiresAt, string _ID, ulong _UserId, string _Username)
    {
        ExpiresAt = _ExpiresAt;
        ID = _ID;
        UserId = _UserId;
        Username = _Username;
    }
}

File: apps/discord-bot/app/src/main/java/discordbot/mongodb/collections/Session.java
package discordbot.mongodb.collections;

import kotlin.ULong;
import java.util.Date;
import org.bson.types.ObjectId;
import org.bson.codecs.pojo.annotations.BsonId;

// TODO: Date probably doesnt work since its old

public class Session {
    @BsonId
    public ObjectId _id;
    /**
     * The date at which the session key expires.
     */
    public Date ExpiresAt;
    /**
     * The primary identifier of the session.
     */
    public String ID;
    /**
     * The Discord ID the session is for.
     */
    public ULong UserId;
    /**
     * The username the session is for.
     */
    public String Username;

    public Session(Date _ExpiresAt, String _ID, ULong _UserId, String _Username) {
        ExpiresAt = _ExpiresAt;
        ID = _ID;
        UserId = _UserId;
        Username = _Username;
    }
}